perception/detected_object_validation/launch/object_position_filter.launch.xml

```diff
@@ -1,12 +1,11 @@
-<?xml version="1.0"?>
+<?xml version="1.0" encoding="UTF-8"?>
 <launch>
+  <arg name="object_position_filter_param_file" default="$(find-pkg-share detected_object_validation)/config/object_position_filter.param.yaml"/>
```

Comment:
Are there any reasons for changing the argument name? Is it to match the name with parameter file name?
Since these arguments could be used by other launch files, it might be better to keep it the same for now. We could fix them when we refactor launch files after parameter work is done. (The same goes for other launch argument names)